### ISSUE TITLE
Add CRISPR repeat and spacer sequences to database

### ISSUE THREAD

**Issue by mbonsma** [goyalsid/phageParser#99]
We want to include the sequences of CRISPR repeats and spacers in our database. Repeat sequences can be useful for comparing CRISPR systems between different species, since species with similar repeat sequences may have some shared CRISPR history. Spacer sequences are the link between viruses and bacteria - since spacers come from viruses or other foreign DNA, studying spacers can shed light on how bacteria and viruses interact. 

One source of this data is the CRISPRdb database, which has detected CRISPR arrays in thousands of species of bacteria and archaea. We will start by adding their data to our database, and going forward we will continue to periodically update our database from theirs. 

---
### The function

**Input:**
- List of CRISPR repeat sequences from CRISPRdb: http://crispr.i2bc.paris-saclay.fr/crispr/BLAST/DR/DRdatabase
- List of CRISPR spacer sequences from CRISPRdb: http://crispr.i2bc.paris-saclay.fr/crispr/BLAST/Spacer/Spacerdatabase

**The function should do the following:**
- Insert each repeat sequence into the **Repeat** table with an auto-generated Primary Key if it isn't already in the table. 
- Insert each spacer sequence into the **Spacer** table with an auto-generated Primary Key if it isn't already in the table. 
- Insert each accession number into the **Organism** table in the **Accession** column. The **OrganismName** column can remain blank for now - this is addressed in another issue.

Each pair of lines in the linked file contains the accession numbers of all the organisms that have a particular sequence followed by the sequence itself. For example, in this block the first line is a list of accession numbers with a postscript, the second is the repeat (GAATCTTCGAGATAGAATTGCAAG). The actual accession number is **just the letters and the first block of numbers**, i.e. `NC_000854`. 

```
>NC_000854_2|NC_000854_4|NC_022521_3|NC_022521_6
GAATCTTCGAGATAGAATTGCAAG
```

**Helpful information:**
Find our SQLite database [here](https://github.com/goyalsid/phageParser/blob/master/crispr.sqlite). Its schema is not in its final form, but it has enough to get us started on rewiring our analysis scripts. Here's what it looks like (courtesy of @dbonsma):

![image](https://cloud.githubusercontent.com/assets/5651370/19663156/c0988fa0-9a08-11e6-964c-6ae6152f648b.png)

**Comment by mbonsma** [comment]
Example data: https://github.com/goyalsid/phageParser/blob/master/data/DRdatabase.txt

**Comment by mbonsma** [comment]
Here's the CRISPRdb: http://crispr.i2bc.paris-saclay.fr/

**Comment by djmoffat** [comment]
Resolved in #105
